cmd/zilstat.in

```diff
@@ -0,0 +1,470 @@
+#!/usr/bin/env @PYTHON_SHEBANG@
+#
+# Print out statistics for all zil stats. This information is
+# available through the zil kstat.
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License, Version 1.0 only
+# (the "License").  You may not use this file except in compliance
+# with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or https://opensource.org/licenses/CDDL-1.0.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# This script must remain compatible with Python 3.6+.
+#
+
+import sys
+import getopt
```

Comment:
nit: using argparse (available in all python3 envs) is much simpler and makes this code clearer

Reply:
For the archives, many early python scripts started out on Solaris that had python 2 without argparse. Later, argparse was added to the default Solaris python installation. So you'll see them using the list of python modules of that era. That era is long dead now with the deprecation of python 2. So it is reasonable to expect python3 and its default list of modules.